projects/vendor/public_api.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Public API Surface of vendor
+ */
+
+export * from './src/yotpo/yotpo.module';
```

Comment:
In order to make this cleaner. For the yotpo folder you should have create an `index.ts` that exports everything that has to be exported by yotpo and export this index file in the public_api.

Reply:
done